### PR TITLE
ADR-1331 filter out non ZADP payments on account in the bta tile call

### PR DESCRIPTION
**Pull request by davidfes**

### CODE REVIEW

File: test/uk/gov/hmrc/alcoholdutyaccount/service/AlcoholDutyServiceSpec.scala [hunk 1]
```diff
@@ -488,6 +488,15 @@ class AlcoholDutyServiceSpec extends SpecBase with TestData {
           result mustBe Success(Some(Payments(Some(Balance(BigDecimal(100), false, Some("X1234567890"))))))
         }
       }
+
+      "filter any payments on account that are not of contractObjectType ZADP" in new SetUp {
```
Comment (yasaracar) [inline test/uk/gov/hmrc/alcoholdutyaccount/service/AlcoholDutyServiceSpec.scala:492]: Add one more test for None please

**Review by yasaracar — CHANGES_REQUESTED** [review]

**Review by yasaracar — APPROVED** [review]
